### PR TITLE
More fixes for typos, grammar, and 404 links

### PR DESCRIPTION
**Pull request by imaustink**
This concludes my review of donejs.com for #916.

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -182,7 +180,7 @@ The DoneJS family of technologies is built by 100s of contributors:
 To become a contributor to DoneJS or its sub-projects, you simply have to:
 
 - [Email](mailto:contact@bitovi.com) the core team expressing your interest.
-- Attend the weekly _DoneJS Contributors_ meeting twice a month. [DoneJS Calendar](https://www.google.com/calendar/embed?src=jupiterjs.com_g27vck36nifbnqrgkctkoanqb4%40group.calendar.google.com&ctz=America/Chicago)
+- Attend the weekly _DoneJS Contributors_ meeting. [DoneJS Calendar](https://www.google.com/calendar/embed?src=jupiterjs.com_g27vck36nifbnqrgkctkoanqb4%40group.calendar.google.com&ctz=America/Chicago)
```
Comment (chasenlehara) [inline docs/about.md]: This is a substantive change that shouldn’t have been made.

File: docs/features.md [hunk 2]
```diff
@@ -757,7 +757,7 @@ for controls. Internally CanJS listens for this element's "removed" event. The "
 * Remove DOM data associated with the element.
 * Remove any template bindings, such as computes bound to text within the template.
 
-CanJS is different from other frameworks in that it will clean up its own memory event when not using the framework to tear down DOM. For example if you were to do:
+CanJS is different from other frameworks in that it will clean up it's own memory event when not using the framework to tear down DOM. For example, if you were to do:
```
Comment (chasenlehara) [inline docs/features.md]: “its” should be possessive, not “it's”

**Review by chasenlehara — CHANGES_REQUESTED** [review]
I love how thorough you were with this. I’ve left a couple comments that need to be resolved.

File: docs/about.md (hunk 1 above)
Comment (imaustink) [reply docs/about.md]: Shoot, this sentence makes sense now! For some reason, it looked like a mistake at the time.

Comment (chasenlehara) [reply docs/about.md]: Please feel free to change it if you can think of a way to make the meaning more clear.

**Comment by imaustink** [comment]
How does it look now?

**Review by chasenlehara — APPROVED** [review]
Looks great!

**Pull request merged**